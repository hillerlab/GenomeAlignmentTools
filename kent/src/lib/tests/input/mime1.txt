Content-type: multipart/form-data, boundary=AaB03x

--AaB03x
content-disposition: form-data; name="field1"

Joe Blow
--AaB03x
content-disposition: form-data; name="pics"; filename="file1.txt"
Content-Type: text/plain

 ... contents of file1.txt ...
--AaB03x--
