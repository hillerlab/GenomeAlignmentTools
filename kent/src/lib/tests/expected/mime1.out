content-type: multipart/form-data, boundary=AaB03x
nested MIME structure

 content-disposition: form-data; name="field1"
 main:[form-data]
 name:[field1]
 size:[8]
 data:[Joe Blow]

 content-disposition: form-data; name="pics"; filename="file1.txt"
 main:[form-data]
 name:[pics]
 filename:[file1.txt]
 content-type: text/plain
 size:[30]
 data:[ ... contents of file1.txt ...]

